tests/python/relay/test_backend_graph_runtime.py

```diff
@@ -184,10 +184,35 @@ def unit_numpy(X, W):
             tvm.testing.assert_allclose(out, ref, rtol=1e-5, atol=1e-5)
 
 
+def test_compile_nested_tuples():
+    x = relay.var("x", shape=(10,))
+    x1 = x + relay.const(1.0)
+    x2 = x1 + relay.const(1.0)
+    x3 = x2 + relay.const(1.0)
+    out = relay.Tuple([x1, relay.Tuple([x2, x3])])
```

Comment:
For example what if x2 is also a tuple.

Reply:
just updated this test to verify two level nesting works.